### ISSUE TITLE
Dockerized chromium support

### ISSUE THREAD

**Issue by MichelJonkman** [beganovich/snappdf#41]
It would be awesome if we could use a docker container for the chromium instance so that it can be kept up-to-date more easily and the dependencies won't cause issues on the server.

Is this already possible or could support be added?

**Comment by beganovich** [comment]
Hi there,

Thanks for reaching out.

What would be a use case for this? Running snappdf outside of containers and Chrome inside of containers?

**Comment by MichelJonkman** [comment]
Yes, exactly, Gotenberg offers a similar solution but this package seems much more lightweight and I don't need more than this package offers

**Comment by beganovich** [comment]
Yeh, Gotenberg is quite different because everything runs in containers.

For now, I suggest setting cron and letting it upgrade Chrome every X

**Comment by eminos** [comment]
I support this FR. It seems much simpler to install and keep updated a dockerized Chrome instance, Browserless for example, than to force a local Chrome/ium install, with all the dependencies, etc... Probably more secure too.
